.github/workflows/special-tests.yml

```diff
@@ -247,3 +262,50 @@ jobs:
       include-success-outputs: ${{ inputs.include-success-outputs }}
       run-coverage: ${{ inputs.run-coverage }}
       debug-resources: ${{ inputs.debug-resources }}
+
+  test-openapi-client:
```

Comment:
Open api tests requires building the airflow python client, so i have added as a separate job here and once it packaged and calling the breeze testing within same job. hope that is okay?